.github/workflows/ping-other-repos.yml

```
@@ -0,0 +1,19 @@
+name: Ping other repos
+
+on:
+  push:
+    branches: [main]
+  repository_dispatch:
+
+jobs:
+  ping-pong:
+    runs-on: ubuntu-latest
+    env:
+      SIDESHOWBARKER_TOKEN: ${{ secrets.SIDESHOWBARKER }}
+    if: ${{ github.respository == 'mdn/browser-compat-data' && env.SIDESHOWBARKER_TOKEN != '' }}
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.SIDESHOWBARKER }}
+          event-type: ping
```

Maybe we ought to still have a payload? Something like…

```suggestion
          event-type: ping
          client-payload: '{"repository": "${{ github.repository }}", "ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
```